src/config.c

```diff
@@ -196,6 +196,24 @@ int loadConfig(char *configFile) {
             // Read next token...
             token = nextConfigToken();
             continue;
+        }
+        else if(strcmp("chroot", token)==0) {
+            // path is in next token
+            token = nextConfigToken();
+            snprintf(commonConfig.chroot, sizeof(commonConfig.chroot), "%s",
+              token);
+            my_log(LOG_DEBUG, 0, "Config: Chroot set to %s",
+              commonConfig.chroot);
+            token = nextConfigToken();
+            continue;
+        }
+        else if(strcmp("user", token)==0) {
+            // username is in next token
+            token = nextConfigToken();
+            snprintf(commonConfig.user, sizeof(commonConfig.user), "%s", token);
```

Comment:
Same for username overflow / truncatenation.

Reply:
Also here, issue has been addressed by checking return value of snprintf().